nnstreamer/tensor_filter/tensor_filter_nntrainer.hh

```diff
@@ -0,0 +1,106 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * GStreamer Tensor_Filter, nntrainer Module
+ * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+ */
+/**
+ * @file	tensor_filter_nntrainer.hh
+ * @date	09 Sept 2020
+ * @brief	nntrainer inference module for tensor_filter gstreamer plugin header
+ * @note  The clas has been exposed from tensor_filter_nntrainer.cc to unittest
+ * @see		http://github.com/nnstreamer/nnstreamer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ * This is the per-NN-framework plugin (tensorflow-lite) for tensor_filter.
```

Comment:
tensorflow-lite?

Reply:
I think the original intention is `e.g) tensorflow-lite`, I will update.

Reply:
Done